scripts/zkai_jstprove_statement_envelope_benchmark.py

```
@@ -0,0 +1,646 @@
+    if not path.is_file():
+        raise JstproveEnvelopeError(f"artifact path is missing: {name}")
+    return path
+
+
+def _artifact_bytes(envelope: dict[str, Any], name: str) -> bytes:
+    if name not in ALLOWED_ARTIFACTS:
+        raise JstproveEnvelopeError(f"unsupported JSTprove artifact reference: {name}")
+    overrides = envelope.get("artifact_overrides", {})
+    if overrides:
+        if not isinstance(overrides, dict):
+            raise JstproveEnvelopeError("artifact_overrides must be an object")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)